### ISSUE TITLE
display action parameters

### ISSUE THREAD

**Issue by SimeonC** [bvaughn/redux-devtools-filterable-log-monitor#22]
Would it be possible to include the action parameters as well like the other LogMonitor?

I think it would just be case of changing FilterableStates.js like this:

``` js

      {expanded &&
        <JSONTree
          data={action}
          labelRenderer={labelRenderer}
          style={{
            marginTop: 0,
            marginBottom: 0,
            marginLeft: 0,
            marginRight: 0
          }}
          theme={theme}
          valueRenderer={valueRenderer}
        />
      }
      <br/>
      {expanded &&
        <JSONTree
          data={filteredState}
          labelRenderer={labelRenderer}
          style={{
            marginTop: 0,
            marginBottom: 0,
            marginLeft: 0,
            marginRight: 0
          }}
          theme={theme}
          valueRenderer={valueRenderer}
        />
      }
```

**Comment by bvaughn** [comment]
Interesting suggestion, @SimeonC. Let me give it a little thought.

**Comment by bvaughn** [comment]
Check it out :)

![untitled screencast - edited](https://cloud.githubusercontent.com/assets/29597/12440009/182bb31c-beec-11e5-8fd0-bdda48e646b2.gif)

**Comment by bvaughn** [comment]
Available as version 0.6. Thank you for the suggestion. :)

**Comment by SimeonC** [comment]
Very nice!! Thanks for the very speedy response!
